CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 0.13.3-dev3
+## 0.13.3-dev4
 
 ### Enhancements
 
 * **Add `strategy` arg value to `_PptxPartitionerOptions`.** This makes this paritioning option available for sub-partitioners to come that may optionally use inference or other expensive operations to improve the partitioning.
+* **Support pluggable sub-partitioner for PPTX Picture shapes.** Use a distinct sub-partitioner for partitioning PPTX Picture (image) shapes and allow the default picture sub-partitioner to be replaced at run-time by one of the user's choosing.
```

Comment:
Is this feature only intended for partitioning images that contain text? Wondering if we will eventually support users plugging in object detection models here so they can have text descriptions of images in a powerpoint.

Reply:
This _specific_ feature supports any of those behaviors. Basically `AbstractPicturePartitioner` takes a `pptx.Picture` object and emits zero or more document elements: `iter_elements(picture: Picture) -> Iterator[Element]`. And, any sub-partitioner that implements this interface (signature) can be registered as the picture subpartitioner. So a subpartitioner that does OCR on the image can be created or one that one that does object detection, or one that does both or does something cool we haven't though of yet :)